com.unity.render-pipelines.core/ShaderLibrary/CommonLighting.hlsl

```diff
@@ -279,7 +279,7 @@ real SphericalCapIntersectionSolidArea(real cosC1, real cosC2, real cosB)
 // ref: Practical Realtime Strategies for Accurate Indirect Occlusion
 // http://blog.selfshadow.com/publications/s2016-shading-course/#course_content
 // Original Cone-Cone method with cosine weighted assumption (p129 s2016_pbs_activision_occlusion)
-real GetSpecularOcclusionFromBentAO(real3 V, real3 bentNormalWS, real3 normalWS, real ambientOcclusion, real roughness)
+real GetSpecularOcclusionFromBentAO_ConeCone(real3 V, real3 bentNormalWS, real3 normalWS, real ambientOcclusion, real roughness)
```

Comment:
Please add this renaming in upgrade guy (in case people where using this function and they will automatically used the new one)